src/styles/routes/share.scss

```
@@ -5,7 +5,7 @@ $cursor-active-override: url('/cursor-click.png'), pointer;
 
 body {
   font-family: $pixel-text-font;
-  background: $accent;
+  background: #016535;
   color: #16161e;
```

```suggestion
  color: #fff;
```

white looks better with background!